index.html

```diff
@@ -7,6 +7,13 @@
     <link href="styles.css" rel="stylesheet" />
     <meta name="author" content="" />
     <meta name="description" content="" />
+    <link rel="icon" type="image/x-icon" href="assets/dharma-wheel.png" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=DM+Sans:opsz@9..40&display=swap"
```

Comment:
So this applies dm sans to the html? Why does it have to be from Google API

Reply:
Easiest method to import font into HTML project